tox.ini

```diff
@@ -0,0 +1,18 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
```

Comment:
No, we have it set to test Python 2.7, 3.4, and 3.5. It's still better to get Travis to test the different versions using tox, so that you get the same test setup when you use Travis vs locally. Plus, tox has a system for installing development dependencies that we should use for our testing. If we get Travis to run the different tox versions, then we have all our test config in one place.

The article I used to create my Travis+tox setup last time is here: https://www.dominicrodger.com/2015/04/14/tox-and-travis-revisited/